### ISSUE TITLE
Failed to include kernel header files on arm64 platform

### ISSUE THREAD

**Issue by Leo-Yan** [iovisor/bpftrace#788]
Hi there,

I try below testing script:

```
root@debian:~/Work# cat path.bt
#include <linux/types.h>
#include <linux/path.h>
#include <linux/dcache.h>

kprobe:vfs_open
{
        printf("open path: %s\n",
                str(((path *)arg0)->dentry->d_name.name));
}
```

It includes Linux kernel header files so that want to trace kernel data structure; I also installed kernel module and headers in /lib/modules.  Finally bpftrace reports error as below:

> root@debian:~/Work# bpftrace path.bt
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/arch/arm64/include/asm/jump_label.h:32:2: error: 'asm goto' constructs are not supported yet
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/arch/arm64/include/asm/jump_label.h:49:2: error: 'asm goto' constructs are not supported yet
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/include/asm-generic/qrwlock.h:65:56: warning: passing 'u32 *' (aka 'unsigned int *') to parameter of type 'int
>  *' converts between pointers to integer types with different sign [-Wpointer-sign]
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/include/linux/atomic-fallback.h:928:46: note: passing argument to parameter 'old' here
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/include/asm-generic/qrwlock.h:92:53: warning: passing 'u32 *' (aka 'unsigned int *') to parameter of type 'int
>  *' converts between pointers to integer types with different sign [-Wpointer-sign]
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/include/linux/atomic-fallback.h:928:46: note: passing argument to parameter 'old' here
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/include/asm-generic/qspinlock.h:65:55: warning: passing 'u32 *' (aka 'unsigned int *') to parameter of type 'i
> nt *' converts between pointers to integer types with different sign [-Wpointer-sign]
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/include/linux/atomic-fallback.h:928:46: note: passing argument to parameter 'old' here
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/include/asm-generic/qspinlock.h:78:52: warning: passing 'u32 *' (aka 'unsigned int *') to parameter of type 'i
> nt *' converts between pointers to integer types with different sign [-Wpointer-sign]
> /lib/modules/5.2.0-rc3-00004-g9b048634b124/source/include/linux/atomic-fallback.h:928:46: note: passing argument to parameter 'old' here

Added more info for bpftrace version and kernel version:
I am using bpftrace version is: bpftrace v0.9-217-g374c and Linux kernel version is: Linux debian 5.2.0-rc3.

Could you suggest for this?  If you think anything I can try, please let me know.  Thanks.

**Comment by mmarchini** [comment]
Hi @Leo-Yan. This issue is likely the same as https://github.com/iovisor/bpftrace/issues/611, which was fixed in #732. Could you try building the latest commit of bpftrace to see if the issue persists?

**Comment by Leo-Yan** [comment]
Hi @mmarchini, I pulled to latest code base and confirmed including header file can work pretty well.

Thanks a lot for the great work!
